core/templates/pages/contributor-dashboard-admin-page/services/contributor-dashboard-admin-backend-api.service.ts

```diff
@@ -159,6 +160,44 @@ export class ContributorDashboardAdminBackendApiService {
       });
     });
   }
+
+  async updateQuestionRightsAsync(
+      username: string,
+      isQuestionSubmitter: boolean,
```

Comment:
userWasGrantedQuestionSubmissionRights
userWasGrantedQuestionReviewRights
userCanCurrentlySubmitQuestions
userCanCurrentlyReviewQuestions

or similar (trying to not start with verb, and trying to show the difference between the two sets of variables).

Reply:
Done